resources/js/components/Courses.vue

```diff
@@ -0,0 +1,218 @@
+<template>
+    <div class="container">
+        <div class="col-12">
+            <div class="card">
+                <div class="card-header">
+                    <h3 class="card-title mb-0">Course List</h3>
+
+                    <div class="card-tools d-flex align-items-center w-75 justify-content-between">
+                        <div class="input-group input-group-sm w-75">
+                            <input type="text" name="table_search" class="form-control float-right" placeholder="Search">
+
+                            <div class="input-group-append">
+                                <button type="submit" class="btn btn-default"><i class="fas fa-search"></i></button>
+                            </div>
+                        </div>
+                        <a @click="newUserModal"><i class="fas fa-book-medical text-success"></i></a>
+                    </div>
+                </div>
+                <!-- /.card-header -->
+                <div class="card-body table-responsive p-0 h-100">
+                    <table class="table table-head-fixed">
+                        <thead>
+                            <tr>
+                                <th>Course ID</th>
+                                <th>Course Name</th>
+                                <th>Course Details</th>
+                                <th>Mastery Score</th>
+                                <th>Owner</th>
+                                <th>Last modified</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr>
+                                <td>1234</td>
+                                <td>Test Course</td>
+                                <td>This is a test Course</td>
+                                <td>100</td>
+                                <td>Sun Asterisk</td>
+                                <td>July 20, 2019</td>
+                            </tr>
+                            <!-- <tr v-for="user, index in users" :key="user.id">
```

Comment:
Are we going to be using this line of code?

Reply:
I will used it for a reference on the next functionality that I will do which is course management.

Reply:
Noted.